readwiseApi.ts

```
@@ -86,25 +84,34 @@ export class ReadwiseApi {
   }
 
   // If lastUpdated or bookID aren't provided, fetch everything.
-  async fetchData(contentType = 'highlights', lastUpdated?: string, bookId?: Number): Promise<Highlight[] | Book[]> {
-    let url = `${API_ENDPOINT}/${contentType}?page_size=${API_PAGE_SIZE}`;
-    if (lastUpdated) url += `&updated__gt=${lastUpdated}`;
-    if (bookId) url += `&book_id=${bookId}`;
-
+  async fetchData(contentType = 'export', lastUpdated?: string, bookId?: Number[]): Promise<Export[]> {
```

Biome (1.9.4)

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)